main.js

```diff
@@ -0,0 +1,107 @@
+// Add event listener 
+// prevent default and const for inputs
+document.getElementById('form-post').addEventListener('submit', function(e) {
+  e.preventDefault();
+
+ 
+  const textFromPost = document.getElementById('text-from-post').value;
+  const writerPosted = document.getElementById('writer-posted').value;
+  
+  postToAdd(textFromPost, writerPosted);
+
+  
+  document.getElementById('text-from-post').value = '';
+  document.getElementById('writer-posted').value = '';
+});
+
+// Fxn to make post add to page html 
+function postToAdd(post, writer) {
+  
+  const containerOfMessages = document.createElement('div');
+  containerOfMessages.classList.add('mb-4', 'border', 'p-3');
+
+  
+  const messagesPost = document.createElement('p');
+  messagesPost.textContent = `${post} - Posted by ${writer}`;
+
+  
+  const withdrawMessagesHere = document.createElement('a');
```

Comment:
bad and not clear const name. Took me a while to understand what this is doing.
Whats wrong with removeMessagesElement/container?